frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -996,7 +1000,7 @@ export const surveyLogic = kea<surveyLogicType>([
             },
         },
         consolidatedSurveyResults: {
-            loadConsolidatedSurveyResults: async (): Promise<ConsolidatedSurveyResults> => {
+            loadConsolidatedSurveyResults: async (limit = 1000): Promise<ConsolidatedSurveyResults> => {
```

Comment:
if we need all, this is a magic number that will not be enough in the future. can we make this query without limits?

Reply:
the limit is actually `10000` [source](https://posthog.com/questions/how-do-i-get-the-next-page-of-results-in-a-hog-ql-result-set-from-the-api) so I'll change do that and add a comment for clarity

Reply:
Should we call the API multiple times until the last page, then? 10k is high, but it's still an arbitrary number that would allow the same problem to happen.
The comment is 2y old so things could have changed

Reply:
Yes, okay to merge and fix since the chances of it happening are much lower now.
I suggest we open a new issue for tracking and explaining the problem. Otherwise, we will run into that again in the future, and we will have some context written down